<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN509708" failures="1" tests="12" name="Test Search in Edge" time="253.961" errors="0" timestamp="2024-04-16T12:17:25 IST">
  <testcase classname="testScenario.TestCaseTravel" name="clickTravelInsurance" time="36.721"/>
  <testcase classname="testScenario.TestCaseTravel" name="clickStartDate" time="19.29"/>
  <testcase classname="testScenario.TestCaseTravel" name="selectAges" time="17.632"/>
  <testcase classname="testScenario.TestCaseTravel" name="enterContactDetais" time="10.929"/>
  <testcase classname="testScenario.TestCaseTravel" name="enterStudentPlans" time="11.776"/>
  <testcase classname="testScenario.TestCaseTravel" name="selectSorts" time="6.758"/>
  <testcase classname="testScenario.TestCaseTravel" name="insurerNameAndPlanPrice" time="1.475"/>
  <testcase classname="testScenario.TestCaseCar" name="sel_NewCar_forInsurance" time="32.679"/>
  <testcase classname="testScenario.TestCaseCar" name="sendData" time="0.142"/>
  <testcase classname="testScenario.TestCaseCar" name="email_Capture_ErrorMessage" time="0.138"/>
  <testcase classname="testScenario.TestCaseCar" name="phoneNo_Capture_ErrorMessage" time="0.061">
    <failure type="java.lang.AssertionError" message="Error Message is not Displayed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Error Message is not Displayed expected [true] but found [false]
at testScenario.TestCaseCar.phoneNo_Capture_ErrorMessage(TestCaseCar.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- phoneNo_Capture_ErrorMessage -->
  <testcase classname="testScenario.TestCaseHealth" name="healthInsuranceMenuItems" time="2.918"/>
</testsuite> <!-- Test Search in Edge -->
